bin/create_img.sh

```diff
@@ -30,15 +30,15 @@ fi
 MOUNTED=1
 sudo mount -o loop $IMG $TMP_DIR
 
-# Create the contens
+# Create the contents
 if [ "$ARCH" == "x86_64" ]; then
 	sudo debootstrap --arch amd64 stable $TMP_DIR
 elif [ "$ARCH" == "i386" ]; then
 	sudo debootstrap --arch i386 stable $TMP_DIR
 elif [ "$ARCH" == "aarch64" ]; then
-	sudo qemu-debootstrap --arch arm64 stable $TMP_DIR
```

Comment:
are you sure qemu-debootstrap is not needed on all systems? Wouldn't it be better to wrap it with a variable

DEBOOTSTRAP=$(which qemu-debootstrap || which debootstrap)

so that we use whatever we find?

Reply:
I think the package has been renamed like `qemu`.

Reply:
FYI:
```Bash
$ ./create_img
...
W: qemu-debootstrap is deprecated. Please use regular debootstrap directly
...
```

Reply:
Sure. But we don't know which system this will work on :)